### DISCUSSION TITLE
Should we use discussions

### DISCUSSION THREAD

**Discussion by clarkegj** [FamilySearch/GEDCOM#68] [Ideas]
Try it out.

**Comment by dthaler** [comment]
I've found that using GEDCOM discussions makes it easy to create issues if needed, mark questions as answered, and up-vote answers to frequently asked questions, while still giving email notifications like a classic mailing list does.

**Reply by dthaler** [reply]
Notes from testing:
* You can reply via email and it will post a response in the discussion.  I did this in https://github.com/FamilySearch/GEDCOM/discussions/67
* As far as I can tell, there is no way to start a brand new discussion purely via email at present.   (Github wouldn't know what discussion category to put it under or what github id to list as the author.)

**Comment by dthaler** [comment]
Comparing this to https://groups.google.com/g/gedcomgeneral ...

Name | Github Discussions | Google Group
---- | ----- | ----
Start new thread via web | Yes | Yes
Start new thread via email | NO | Yes
Respond via web | Yes | Yes
Respond via email | Yes | Yes
Mark question as answered | Yes | NO
Categorize threads | Yes | NO
Create github issue from thread | Yes | NO
Link thread with github issue | Yes | NO
Lock thread | Yes | NO
Enumerate members of list | Yes | Yes
Up-vote replies | Yes | NO


 to https://groups.google.com/g/gedcomgeneral/c/S32eeVs16dQ
